src/gc/gccommon.cpp

```diff
@@ -15,10 +15,10 @@
 #include "gc.h"
 
 #ifdef FEATURE_SVR_GC
-SVAL_IMPL_INIT(uint32_t,GCHeap,gcHeapType,GCHeap::GC_HEAP_INVALID);
+SVAL_IMPL_INIT(uint32_t,IGCHeap,gcHeapType,IGCHeap::GC_HEAP_INVALID);
 #endif // FEATURE_SVR_GC
 
-GPTR_IMPL(GCHeap,g_pGCHeap);
+IGCHeapInternal* g_theGcHeap;
```

Comment:
Ah, got it, then I believe it is ok.

---
In reply to: [77906349](https://github.com/dotnet/coreclr/pull/6764#discussion_r77906349) [](ancestors = 77906349)